### ISSUE TITLE
Should warn on missing ":" for some syntaxes like "..note"

### ISSUE THREAD

**Issue by sylvestre** [myint/rstcheck#68]
For example, if you write
```
..note:

     foo bar
```

Sphinx won't show anything because of a missing `:`

rstcheck won't complain while it could detect that `..note:$` isn't correct.

The correct syntax being
```
..note::

     foo bar
```

**Comment by leungyukshing** [comment]
Agreed! This tool should raise some error if the syntax is incorrect.

**Comment by Cielquan** [comment]
I guess `..note:` is interpreted as a comment. Therefore possible false positives are possible.
But I think the value of having this check is bigger than having rare false positives if someone writes a comment like this.

Are you willing to implement this feature?

**Comment by Cielquan** [comment]
I looked deeper into this matter and here are my findings:

1. You need to add a space between the directive double dot and the directives name: `..note::` ->`.. note::` else docutils will not recognize it as an directive but rather plain text.
2. I am indeed right. docutils does see `.. note:` as a comment because of the missing double colon and therefore this is no error.

```rst
.. note:

     foo bar

.. note::

     foo bar
```
results in this html
```html
<!-- note:

foo bar -->
<aside class="admonition note">
<p class="admonition-title">Note</p>
<p>foo bar</p>
</aside>
```

### Conclusion

This issue would be an upstream issue with docutils. So you can open a ticket over [here](https://sourceforge.net/p/docutils/_list/tickets).

Or you can create a regex rule if and when https://github.com/myint/rstcheck/issues/80#issuecomment-1102849164 is implemented.
